### PR TITLE
Port fixes to release 2.0

### PR DESCRIPTION
**Pull request by karajas**
@rakeshsinghranchi @chcosta @eerhardt @weshaggard

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -381,6 +381,36 @@
...
 
+  <!-- Use actual publishable (non-dummy) package name produced by the build system for this RID -->
+  <PropertyGroup Condition="'$(TargetRid)' != ''">
+    <PackageTargetRid>$(TargetRid)</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'osx.10.11-x64'">osx.10.10-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'rhel.7.0-x64'">rhel.7-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'rhel.7.1-x64'">rhel.7-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'rhel.7.2-x64'">rhel.7-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'rhel.7.3-x64'">rhel.7-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'rhel.7.2-x64'">rhel.7-x64</PackageTargetRid>
```
Comment (omajid) [inline dir.props]: There's two `rhel.7.2` in this list. I take it the second one should be `7.4`?

File: dir.props [hunk 2]
```diff
@@ -381,6 +381,36 @@
     <CombinedInstallerExtension Condition="'$(OSGroup)' != 'Windows_NT'">$(InstallerExtension)</CombinedInstallerExtension>    
   </PropertyGroup>
 
+  <!-- Use actual publishable (non-dummy) package name produced by the build system for this RID -->
+  <PropertyGroup Condition="'$(TargetRid)' != ''">
+    <PackageTargetRid>$(TargetRid)</PackageTargetRid>
+    <PackageTargetRid Condition="'$(TargetRid)' == 'osx.10.11-x64'">osx.10.10-x64</PackageTargetRid>
```
Comment (gkhanna79) [inline dir.props]: There is no 10.10 or 10.11 - how is this getting specified?

File: dir.props (hunk 1 above)
Comment (gkhanna79) [inline dir.props]: Is this for BFS?

Comment (karajas) [inline dir.props]: This is for packaging consistency @chcosta

File: dir.props (hunk 2 above)
Comment (karajas) [inline dir.props]: Think this is residue from the old world. @chcosta ?

**Comment by karajas** [comment]
Merging this in and kicking off an official run. Will be on point to deal with any fallout.

**Comment by gkhanna79** [comment]
Please email shiproom about this.

**Pull request merged**